frontend/src/app/[organisation]/bemanning/api/weeklyWorkHours/route.ts

```diff
@@ -0,0 +1,14 @@
+import { fetchWithToken } from "@/data/fetchWithToken";
+import { NextResponse } from "next/server";
+
+export async function GET(request: Request) {
+  const { searchParams } = new URL(request.url);
+  const organisationName = searchParams.get("organisationName") || "";
+
+  const weeklyWorkHours =
+    (await fetchWithToken<number>(
+      `organisations/${organisationName}/weeklyWorkHours`,
+    )) ?? [];
+
```

Comment:
1: Viktig å skille på Org. ID (1), Org Navn (Variant AS) og Org UrlKey (variant-norge)
2: Tror vi kan hente Org Url Key fra URL og ikke søkeparamentre, ref kallet her: ``/${organisationName}/bemanning/api/weeklyWorkHours?organisationName=${organisationName}`,`

Reply:
Godt poeng! Gjerne kom med forslag om du har en fremgangsmåte som ikke bruker søkeparameter 😅 Hvis vi bruker split, er jeg litt bekymret for hva som skjer hvis URL endrer seg i fremtiden, men strukturen på denne er kanskje satt?